ui/components/src/model/units/constants.ts

```diff
@@ -13,4 +13,7 @@
 
 // Common constants needed across individual unit groups and the overall
 // combined units.
-export const DEFAULT_DECIMAL_PLACES = 2;
+export const MAX_SIGNIFICANT_DIGITS = 3;
+
+export const DECIMAL_PLACES_MIN = 0;
+export const DECIMAL_PLACES_MAX = 20;
```

Comment:
We don't need `DEFAULT_DECIMAL_PLACES` anymore

We do need a constant to represent the default # of significant figures to use

The other two constants are for the `decimal_places` input with `type="number"`

Reply:
Per our conversation with Mohit this morning, I think bringing this number down a bit would be a good idea. We can always increase it later if we get requests.